studybuddies/src/app/services/salas.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { SalasService } from './salas.service';
+
+describe('SalasService', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
describe("SalasService", () => {
```